src/System.Management.Automation/help/CommandHelpProvider.cs

```diff
@@ -521,6 +523,13 @@ private string GetHelpFile(string helpFile, CmdletInfo cmdletInfo)
                 {
                     searchPaths.Add(GetDefaultShellSearchPath());
                     searchPaths.Add(GetCmdletAssemblyPath(cmdletInfo));
+
+                    // in case of Microsoft.PowerShell.Core module, cmdletInfo.ModuleName is null.
+                    // if cmdletInfo.ModuleName is null, Path.Combine return HelpUtils.GetUserHomeHelpSearchPath()
+                    // So we will be searching under HelpUtils.GetUserHomeHelpSearchPath()
+                    // The search is done in order of the searchPath list. We keep the userHelp
```

Comment:
nit: you can use the capital letters at the beginning of the sentence and punctuation:

> // In case of Microsoft.PowerShell.Core module, cmdletInfo.ModuleName is null.
> // If cmdletInfo.ModuleName is null, Path.Combine returns HelpUtils.GetUserHomeHelpSearchPath(),
> // so we will be searching under HelpUtils.GetUserHomeHelpSearchPath().
> // The search is done in order of the searchPath list.

Reply:
Fixed.